grakn-graph/src/main/java/ai/grakn/graph/internal/GraknComputerImpl.java

```diff
@@ -54,9 +56,13 @@
     private final Class<? extends GraphComputer> graphComputerClass;
     private GraphComputer graphComputer = null;
 
-    public GraknComputerImpl(Graph graph, String graphComputerType) {
+    public GraknComputerImpl(Graph graph) {
         this.graph = graph;
-        this.graphComputerClass = getGraphComputerClass(graphComputerType);
+        if(graph instanceof TinkerGraph){
+            graphComputerClass = TinkerGraphComputer.class;
+        } else {
+            graphComputerClass = GraknSparkComputer.class;
+        }
```

Comment:
Got it! I see we only do that in one case: `ResourceDeduplicationTask`, so it might still be worth looking into.